src/tests/specs/setupTests.ts

```diff
@@ -0,0 +1,39 @@
+export {};
+
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toContainStringContaining(expected: string): CustomMatcherResult;
+    }
+  }
+}
+
+expect.extend({
+  toContainStringContaining(
```

Comment:
Useful matcher to check whether the error / warning arrays contained the offending messages.